src/main/java/org/openhab/tools/analysis/checkstyle/ExportInternalPackageCheck.java

```diff
@@ -24,29 +24,29 @@
  */
 public class ExportInternalPackageCheck extends AbstractStaticCheck {
 
-    public final static String MANIFEST_EXTENSTION = "MF";
+    public final static String MANIFEST_EXTENSION = "MF";
```

Comment:
No, I don't want separate PR, I just missed the spelling error. Thanks for fixing them!